src/main/webapp/src/theme.scss

```diff
@@ -0,0 +1,198 @@
+/* Color scheme for our app -- primary #58d68d, secondary #005c80 */
+@import '~@angular/material/theming';
+
+// Theme generated by https://materialtheme.arcsine.dev/
```

Comment:
are there any hand-edits or was this taken directly from the theme generator?

Reply:
I removed parts that specified fonts.

Reply:
I will annotate this.